extensions/ql-vscode/test/vscode-tests/ensureCli.ts

```diff
@@ -38,7 +39,7 @@ const _10MB = _1MB * 10;
 
 // CLI version to test. Hard code the latest as default. And be sure
 // to update the env if it is not otherwise set.
-const CLI_VERSION = process.env.CLI_VERSION || "v2.11.6";
+const CLI_VERSION = process.env.CLI_VERSION || data[0];
```

Comment:
Minor: we could make this variable name more descriptive 😄 

```suggestion
const CLI_VERSION = process.env.CLI_VERSION || supportedCliVersions[0];
```